docs/usage.md

```diff
@@ -30,3 +30,8 @@ Read the [Git manual](https://git-scm.com/docs/gitcredentials#_custom_helpers) a
 Set your user-level Git configuration (`~/.gitconfig`) to use GCM Core. If you pass
 `--system` to these commands, they act on the system-level Git configuration
 (`/etc/gitconfig`) instead.
+
+### azure-repos
```

Comment:
Would be nice to have additional documentation about how manual binding works and any automated binding behaviors as a result of "simply" signing in.

Reply:
Right. I can work on this.. will need some expert editing to make it understandable to humans!

Reply:
@vtbassmatt I've added a new document outlining the differences between PATs and Azure tokens, as well as how users are bound (and how advanced users can play with them).